home_page.php

```diff
@@ -59,8 +59,8 @@
 			</header>
 			
 			<!-- Section 1: Background image display -->
-			<div class="section1"style = "background-color:gray;">
```

Comment:
This background color was also given as a placeholder for the section 1 image. It was just to define its section so that when I reached the images & graphics stage, I could remove this and add the image